lib/App/DuckPAN/Web.pm

```diff
@@ -539,6 +541,19 @@ sub request {
 	return $response;
 }
 
+
+sub run_on_ready_script {
+	my ($str) = @_;
+
+	# any unescaped script tags at this stage will XSS the page - so strip them
+	$str =~ s|</?script>||gi;
```

Comment:
I guess `<script/>` won't cause any problems as browsers won't expand it out? It should nest even if it did, but maybe worth a test?